server/filestore.go

```diff
@@ -7946,7 +7947,11 @@ func (fs *fileStore) setSyncTimer() {
 	if fs.syncTmr != nil {
 		fs.syncTmr.Reset(fs.fcfg.SyncInterval)
 	} else {
-		fs.syncTmr = time.AfterFunc(fs.fcfg.SyncInterval, fs.syncBlocks)
+		// First time this fires will be any time up to the fs.fcfg.SyncInterval,
+		// so that different stores are spread out, rather than having many of
+		// them trying to all sync at once, causing blips and contending dios.
+		start := time.Duration(mrand.Int64N(int64(fs.fcfg.SyncInterval)))
```

Comment:
This could return 0, should we add a fixed floor?

Reply:
Sure, will do.